app/View.ts

```diff
@@ -151,18 +151,18 @@ class View {
             }
           }
 
-        Chart.defaults.global.defaultFontColor = "black";
-        Chart.defaults.global.defaultFontFamily = "Roboto Condensed";
+        // Chart.defaults.global.defaultFontColor = "black";
```

Comment:
These actually do work, but I need to figure out how to make the compiler aware of external dependencies

Reply:
Ah, cool. I meant to uncomment those lines but forgot to, I assumed you had them working on your end.